Omargh/main.py

```diff
@@ -0,0 +1,46 @@
+from fastapi import FastAPI,HTTPException, Path
+from typing import Dict
+from modules import Task, UrgentTask,TaskCreate
+from uuid import UUID, uuid4 
+from taskmanager import TaskManager
+
+app = FastAPI()
+# task_manager is made from taskmanager class that work with dictionary
+task_manager = TaskManager()
+""" Creating a Task"""
+@app.post("/tasks/")
+async def create_task(task_data: TaskCreate, is_urgent: bool = False):
```

Comment:
I suggest using the priority in the body of the base model 
if the Priority not nil -> urgent task
if Priority nil -> normal task